test/abstracts.md

```diff
@@ -8,6 +8,39 @@ It's rather a specification than current implementation,
 but the goal is to optimize such specification as much as possible
 and achieve best performance on top of that.
 
+
+
+### Definitions
+
+`hyperHTML` is a function tag for template literals.
+
+Its `statics` argument is a unique templateObject
+http://www.ecma-international.org/ecma-262/6.0/#sec-gettemplateobject
+
+Its `context` MUST be a DOM Element.
+
+`hyperHTML.bind(context)` creates a new tag for such `context`.
+
+A list of `paths` is an array of instructions used to address a specific target node.
+Each instruction also carry the kind of update operation the target needs.
```

Comment:
s/update operation the target needs/update operation to be performed on the target node

Can you add what each number means via a comment? Then one can just look at a path and literally see what it's going to do.